### DISCUSSION TITLE
Welcome to flixy Discussions!

### DISCUSSION THREAD

**Discussion by SKbarbon** [SKbarbon/flixy#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Good luck, me :)

**Comment by nervosajj80** [comment]
Thank you for contributing this library. It's good to see pythonista finally moving into the future after such a lengthy hiatus. Flixy looks promising and I look forward to learning all about it. 
 My question is this; is the newer api of  flixxy user friendly in regards to the Scene module? How easily can flixy be integrated into a scene or, visa versa, scene into flixy? What would be the steps to do so?

**Reply by SKbarbon** [reply]
Thanks for your support!!
The current goal is to make the UI scripting better. Then yes The second goal is to build a sub-module called "flixy.game" that have uses pythonista scene library as a back-end.

Just be wait for it 🌝.

**Reply by nervosajj80** [reply]
That sounds interesting. So, if I understand correctly, it would be a submodule of Flixy that incorporates scene and ui and effectively abstracts away the need to even worry with the interactions to implement one using the other. In other words; Flixy will handle all of the two modules' interactions internally using one new, quite handy API? That's pretty user-friendly indeed.  
I can't wait to see this endeavor unfold. If you need a beta tester or perhaps just consultstion or some fresh eyes for a review, I'd be happy to do whatever I can. I think it could be a useful project. Once again, thank you!

**Reply by SKbarbon** [reply]
Thanks!

**Comment by ikiziki** [comment]
Forgive my ignorance here, I'm completely new to all this. 

If I write an app using this framework will it also run on other platforms (windows, Linux, Mac)? Or is it iOS specific?

**Reply by SKbarbon** [reply]
Hello @AltKrypto ,
Honestly, that wasn’t on the plan. But it sounds interesting!!.
I will put that on the goal!, convert pythonista flixy into macOS, windows and linux application!.

**Reply by ikiziki** [reply]
> Hello @AltKrypto ,
> 
> Honestly, that wasn’t on the plan. But it sounds interesting!!.
> 
> I will put that on the goal!, convert pythonista flixy into macOS, windows and linux application!.

I've been on the hunt for a cross platform GUI package that plays well with Pythonista, so far no luck.

**Reply by SKbarbon** [reply]
@AltKrypto Hey!
I lessened to your suggestion..
Now its done! Pythonista-flixy now support desktop!!
And..?
Now you can convert pythonista flixy scripts into macOS or windows app!!

Check the announcement here:
https://github.com/SKbarbon/flixy/discussions/5

And check the README to find how to install it:
https://github.com/SKbarbon/flixy/

🌝🥂

**Reply by ikiziki** [reply]
> @AltKrypto Hey!
> I lessened to your suggestion..
> Now its done! Pythonista-flixy now support desktop!!
> And..?
> Now you can convert pythonista flixy scripts into macOS or windows app!!
> 
> Check the announcement here:
> https://github.com/SKbarbon/flixy/discussions/5
> 
> And check the README to find how to install it:
> https://github.com/SKbarbon/flixy/
> 
> 🌝🥂

Amazing, once I'm back home and stationary I'll give it a try!

**Reply by SKbarbon** [reply]
> > @AltKrypto Hey!
> > I lessened to your suggestion..
> > Now its done! Pythonista-flixy now support desktop!!
> > And..?
> > Now you can convert pythonista flixy scripts into macOS or windows app!!
> > Check the announcement here:
> > #5
> > And check the README to find how to install it:
> > https://github.com/SKbarbon/flixy/
> > 🌝🥂
> 
> Amazing, once I'm back home and stationary I'll give it a try!

@AltKrypto good luck, if you find any issue please submit it https://github.com/SKbarbon/flixy/issues
